modules/osd/label/index.ts

```diff
@@ -15,19 +16,29 @@ export const OSDLabel = (ort: OSDOrientation) => {
             vpack: "center",
             setup: self => {
                 self.hook(brightness, () => {
+                    self.class_names = self.class_names.filter(c => c !== "overflow");
                     self.label = `${Math.round(brightness.screen * 100)}`;
                 }, "notify::screen")
                 self.hook(brightness, () => {
+                    self.class_names = self.class_names.filter(c => c !== "overflow");
                     self.label = `${Math.round(brightness.kbd * 100)}`;
                 }, "notify::kbd")
                 self.hook(audio.microphone, () => {
                     self.toggleClassName("overflow", audio.microphone.volume > 1)
                     self.label = `${Math.round(audio.microphone.volume * 100)}`;
                 }, "notify::volume")
+                self.hook(audio.microphone, () => {
+                    self.toggleClassName("overflow", audio.microphone.volume > 1 && (!options.theme.osd.muted_zero.value || !(audio.microphone.is_muted !== false)));
+                    self.label = `${options.theme.osd.muted_zero.value && audio.microphone.is_muted !== false ? 0 : Math.round(audio.microphone.volume * 100)}`;
```

Comment:
```suggestion
                        self.value = (options.theme.osd.muted_zero.value && audio.microphone.is_muted !== false) ? 0
                            : audio.microphone.volume <= 1 ? audio.microphone.volume
                                : audio.microphone.volume - 1;

```
Let's make this more readable